js/app.js

```diff
@@ -1 +1,364 @@
-console.log('Hello frontend');
+let options = ["x", "o"];
+let choice = options[0];
+
+//collecting data on starting choice
+//to see which box is clicked
+if(document.querySelector('input[name="choice"]')) {
+    document.querySelectorAll('input[name="choice"]').forEach((element) => {
+    element.addEventListener("change", function(event) {
+        let firstChoice = event.target.value;
+        if (firstChoice === "0") {
+            choice = options[0];
+            console.log(options[0]);
+            showWinner1.innerText = "turn: x";
+            
+        } 
+        if (firstChoice === "1") {
+            choice = options[1];
+            console.log(options[1]);
+            showWinner1.innerText = "turn: o";
+        }
+    });
+    }); 
+}
+
+//if box 1 is clicked
```

Comment:
These lines from 26 through 131 are a great candidate for refactoring and making your code DRYer.  You saw some examples of that earlier in the presentations, so hopefully that gives you an idea of what I mean and some idea of how to implement it.